core/src/main/java/org/mapfish/print/map/tiled/TileLoaderTask.java

```diff
@@ -333,6 +349,11 @@ public PlaceHolderImageTask(final BufferedImage placeholderImage, final int tile
         protected Tile compute() {
             return new Tile(this.placeholderImage, getTileIndexX(), getTileIndexY());
         }
+
+        @Override
+        public Tile call() throws Exception {
```

Comment:
Are you sure you need to override this method? The new implementation is the same as the one inherited from TileTask.